Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+
+        public static void Demo()
+        {
+
+            EnterUserData();
+            EnterUserData();
+
+        }
+
+        public static void EnterUserData()
+        {
+
+            string firstName = Lesson3.PromptUserString("First name: ");
+            string lastName = Lesson3.PromptUserString("Last name: ");
+            int age = Lesson3.PromptUserInt("Age: ");
+            float height = Lesson3.PromptUserFloat("Height (in m): ");
+            float weight = Lesson3.PromptUserFloat("Weight (in kg): ");                      
+            float bmi = Lesson3.CalculateBmi(weight, height);
+
+            Console.WriteLine(firstName + " " + lastName + " is " + age + " years old, their weight is " + weight + "kg and their height is " + height + "m. Their BMI is " + bmi + ".");
+            Console.WriteLine();
+        }
+
+
+        public static string PromptUserString(string message)
+        {
+            Console.WriteLine(message);
+            string name = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(name))
+            {
+                Console.Write("Name cannot be empty.\n");
+                return "-";
+            }
+
+            return name;
+
+        }
+
+         public static int PromptUserInt(string message)
+        { 
+            // Input is not a number -> return -1.
+            Console.WriteLine(message);
+            int num = int.Parse(Console.ReadLine());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'num'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4910191)